Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/FeedOpenCTI_v2.yml

```diff
@@ -0,0 +1,556 @@
+- display: Base URL
+  hidden: false
+  name: base_url
+  required: true
+  type: 0
+- display: API Key
+  hidden: false
+  name: apikey
+  required: true
+  type: 4
+- additionalinfo: 'The indicator types to fetch. Out of the box indicator types supported
+    in XSOAR are: "Account", "Domain", "Email", "File", "Host", "IP", "IPv6", "Registry Key",
+    and "URL". The rest will not cause automatic indicator creation in XSOAR.
+    The default is "ALL".'
+  defaultvalue: ALL
+  display: Indicators Type to fetch
+  name: indicator_types
+  options:
+  - ALL
+  - Account
+  - Domain
+  - Email
+  - File
+  - Host
+  - IP
+  - IPv6
+  - Registry Key
+  - URL
+  required: true
+  type: 16
+- defaultvalue: '500'
+  display: Max. indicators per fetch
+  hidden: false
+  name: max_indicator_to_fetch
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  display: Indicator Reputation
+  name: feedReputation
+  defaultvalue: Good
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: F - Reliability cannot be judged
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+    fetched from the feed
+  defaultvalue: GREEN
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+- defaultvalue: indicatorType
+  display: ''
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: '20160'
+  display: ''
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  required: false
+  type: 0
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Ingest indicator feeds from OpenCTI. Works with OpenCTI v4 instances.
+display: OpenCTI Feed v2
```

Comment:
i think `v2` is confusing here because this integration is actually used to work with v4 of opencti